bundles/core/org.eclipse.smarthome.core.id/src/main/java/org/eclipse/smarthome/core/id/internal/UUIDResource.java

```diff
@@ -28,6 +29,7 @@
  */
 @Path(UUIDResource.PATH_UUID)
 @Api(value = UUIDResource.PATH_UUID)
+@RolesAllowed({ "administrator" })
```

Comment:
Shall we define a default set of roles as constants in org.eclipse.smarthome.core.auth.Roles that should be used throughout the ESH code?